### DISCUSSION TITLE
windows subsystem for android and Trading applications like Binance bybit apps

### DISCUSSION THREAD

**Discussion by nassirok** [community/community#144343] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello, I have a question and I hope you can answer it. I have installed windows subsystem for android v 2309 on Windows 11 and everything is fine, All applications are working except for Trading applications like BINANCE and BYBIT and BITGATE and OCX  , It doesn't open and gets stuck on the logo as shown in the attached images. Is there a solution, please?
![image](https://github.com/user-attachments/assets/34d9bfc1-04c1-4d3e-b4be-35053342f599)
![image](https://github.com/user-attachments/assets/031ff3da-fa53-4d20-b145-b7b1b85d85d7)

**Comment by samus-aran** [comment]
Hi @nassirok ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on installation. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
